umap/distances.py

```diff
@@ -42,7 +42,7 @@ def euclidean_grad(x, y):
     for i in range(x.shape[0]):
         result += (x[i] - y[i]) ** 2
     d = np.sqrt(result)
-    grad = (x-y)/(1e-6 + d)
+    grad = (x - y) / (1e-6 + d)
```

Comment:
much of this is just autoformatting the file, according to the style rules in `.idea/` already in the repo